babun-core/tools/procps.sh

```diff
@@ -13,23 +13,23 @@ function check_only_one_running {
     if mkdir /var/lock/$proc_name 2>/dev/null ; then
         trap 'rm -rf "/var/lock/$proc_name"; exit $?' INT TERM EXIT
     else
-        echo "You can only run one instance of $proc_name. Exiting!";
+        echo "You can run only one instance of $proc_name. Close all other instances of $proc_name or clear the /var/lock/ directory if you are sure there are no other instances currently running."
         exit 1;
```

Comment:
Thx!  There may be other files in the the /var/lock/ directory though.
It should be more like:
`Close all other instances of $proc_name or delete the /var/lock/$proc_name file if you are sure there are no other instances currently running.`